tests/hexahedron_element_test.cc

```diff
@@ -934,6 +934,10 @@ TEST_CASE("Hexahedron elements are checked", "[hex][element][3D]") {
         for (unsigned j = 0; j < indices.cols(); ++j)
           REQUIRE(check_indices(j) == indices(i, j));
       }
+
+      // Check number of faces
+      const unsigned check_nfaces = 6;
+      REQUIRE(check_nfaces == hex->nfaces());
```

Comment:
You check if LHS == 6, not other way around. no need to define `check_nfaces` just add the comment in the line above.